### PR TITLE
feat : Added Quickstarts for vert.x eventbus and extensions - New Relic Labs

### PR DESCRIPTION
**Pull request by gsidhwani-nr**
…ic Labs

# Summary

Added Quickstarts for vert.x eventbus and extensions - New Relic Labs
Pre-Existing Java Dashboards, Alerts  are included in Quickstart.

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X ] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/java/vertx-eventbus/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+id:
```
Comment (pkudikyala) [inline quickstarts/java/vertx-eventbus/config.yml:1]:
```suggestion

```

File: quickstarts/java/vertx-extensions/config.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+id:
```
Comment (pkudikyala) [inline quickstarts/java/vertx-extensions/config.yml:1]:
```suggestion

```

**Review by pkudikyala — CHANGES_REQUESTED** [review]
Hi @gsidhwani-nr ,
Could you make the review changes.

**Comment by gsidhwani-nr** [comment]
The requested changes have been committed.

**Review by pkudikyala — APPROVED** [review]
LGTM!

**Pull request merged**